### PR TITLE
Extend changelog for 2.0.0

### PR DESCRIPTION
**Pull request by mxmehl**
Explain the cause of some linting errors.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -40,6 +40,25 @@ CLI command and its behaviour. There are no guarantees of stability for the
 
 ## 2.0.0 - 2023-06-21
 
+As you can see, this major release is packed with a lot of changes, and some of
+them have been requested by a lot of people. Probably most notably is the
```
Comment (carmenbianca) [inline CHANGELOG.md:44]: notably → notable

File: CHANGELOG.md [hunk 2]
```diff
@@ -40,6 +40,25 @@ CLI command and its behaviour. There are no guarantees of stability for the
...
+`--json` flag for the linter, easing the parsing of its output.
+
+**Possibly breaking change**: In some cases, the linting of some projects now
+fails even though they didn't in version `1.x`. This is because we finally
+defined a strict order of precedence of the different source for licensing and
+copyright information. In the old version, of a file was described by more than
```
Comment (carmenbianca) [inline CHANGELOG.md:50]: of → if

File: CHANGELOG.md [hunk 3]
```diff
@@ -40,6 +40,25 @@ CLI command and its behaviour. There are no guarantees of stability for the
...
+one source, the information has been merged. This often lead to unexpected
```
Comment (carmenbianca) [inline CHANGELOG.md:51]: has been → would be

lead → led

File: CHANGELOG.md [hunk 4]
```diff
@@ -40,6 +40,25 @@ CLI command and its behaviour. There are no guarantees of stability for the
...
+results. Therefore, in line with the newly released REUSE specification version
+3.1, the [order of precedence](https://reuse.software/spec/#order-of-precedence)
+has been defined and the merging has been disabled.
+
+We are aware that this will break the REUSE compliance of some projects, but we
+are confident that it's better this way to really make licensing and copyright
+transparent and understandable. Please take the time to fix the uncertainties in
```
Comment (carmenbianca) [inline CHANGELOG.md:58]: this way to [...] → this way to strictly enforce a single source of truth for copyright and licensing information.

**Review by carmenbianca — CHANGES_REQUESTED** [review]
Another suggestion:

The bullet point '- More [files/file types] are recognised' is duplicated. Let's remove one of them :)

File: CHANGELOG.md (hunk 1 above)
Comment (mxmehl) [reply CHANGELOG.md:44]:
```suggestion
them have been requested by a lot of people. Probably most notable is the
```

**Comment by carmenbianca** [comment]
Closing this in favour of #777

**Pull request closed without merging**